tests/test_model_methods.py

```diff
@@ -255,6 +257,37 @@ def test_setup_reservoirs(source, tmpdir, example_wflow_model):
 number of reservoirs in model area"
 
 
+def test_setup_ksathorfrac(tmpdir, example_wflow_model):
+    # Read the modeldata
+    model = "wflow"
+    example_wflow_model.read()
+    # Create dummy ksat data
+    da = full_like(example_wflow_model.grid["KsatHorFrac"])
+    data = np.zeros(da.shape)
+    for x, y in product(*[range(item) for item in da.shape]):
+        data[x, y] = 750 - ((x + y) ** 0.4 * 114.07373)
+    da.values = data
+
+    # Set the output directory
+    destination = str(tmpdir.join(model))
+    example_wflow_model.set_root(destination, mode="w")
+
+    # Build the map
+    example_wflow_model.setup_ksathorfrac(
+        ksat_fn=da,
+    )
+    # Write and read the map
+    example_wflow_model.write_grid()
+    example_wflow_model.read_grid()
```

Comment:
Don't think you need these lines in your test, do you?

Reply:
Maybe a bit of legacy now, I wasnt able to read the results in Python previously without doing this. I shall check that.

Reply:
I found my original reason, the data is clipped when written and I wanted to check based on that.